admin_guide/topics/testing-auto-scaler.adoc

```diff
@@ -0,0 +1,154 @@
+// Module included in the following assemblies:
+//
+// * admin_guide/cluster-autoscaler.adoc
+
+[id='testing-AWS-cluster-auto-scaler_{context}']
+= Testing the auto-scaler
+
+After you add the auto-scaler to your Amazon Web Services (AWS) cluster, you can
+confirm that the auto-scaler works..
+
+.Prerequisites
+
+* You added the auto-scaler to your {product-title} cluster that runs on AWS.
+
+.Procedure
+
+. Create the *_scale-up.yaml_* file that contains the deployment configuration
+to test auto-scaling:
++
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: scale-up
+  labels:
+    app: scale-up
+spec:
+  replicas: 20 <1>
+  selector:
+    matchLabels:
+      app: scale-up
+  template:
+    metadata:
+      labels:
+        app: scale-up
+    spec:
+      containers:
+      - name: origin-base
+        image: openshift/origin-base
+        resources:
+          requests:
+            memory: 2Gi
+        command:
+        - /bin/sh
+        - "-c"
+        - "echo 'this should be in the logs' && sleep 86400"
+      terminationGracePeriodSeconds: 0
+----
++
+<1> This deployment specifies 20 replicas, but the initial size of the cluster
+cannot run all of the pods without first increasing the number of compute nodes.
```

Comment:
@kalexand-rh 
_ without first increasing the number of compute nodes_ which is done automatically by the auto scaler?